fbw-a380x/src/wasm/fbw_a380/src/interface/SimConnectInterface.cpp

```diff
@@ -321,7 +321,7 @@ bool SimConnectInterface::prepareSimInputSimConnectDataDefinitions() {
   result &= addInputDataDefinition(hSimConnect, 0, Events::AP_MASTER, "AP_MASTER", true);
   result &= addInputDataDefinition(hSimConnect, 0, Events::AUTOPILOT_OFF, "AUTOPILOT_OFF", false);
   result &= addInputDataDefinition(hSimConnect, 0, Events::AUTOPILOT_ON, "AUTOPILOT_ON", true);
-  result &= addInputDataDefinition(hSimConnect, 0, Events::AUTOPILOT_DISENGAGE_SET, "AUTOPILOT_DISENGAGE_SET", true);
+  result &= addInputDataDefinition(hSimConnect, 0, Events::AUTOPILOT_DISENGAGE_SET, "AUTOPILOT_DISENGAGE_SET", false);
```

Comment:
This is quite a risky change, as the event will now go to the sim AP. It might be better to send an event to the FWC from within the handler in this interface.

Reply:
Understood, this will stay masked an I'll emit K:A32NX.AUTOPILOT_DISENGAGE